lib/rules/media-query-no-invalid/index.mjs

```diff
@@ -143,6 +158,31 @@ const rule = (primary) => {
 	};
 };
 
+/**
+ * @param {import('@csstools/media-query-list-parser').MediaQuery|import('@csstools/media-query-list-parser').GeneralEnclosed} node
+ * @param {Object|undefined} secondaryOptions
+ * @returns {boolean}
+ */
+function shouldIgnoreNode(node, secondaryOptions) {
+	let ignored = false;
+
+	if (secondaryOptions && 'ignoreFunctions' in secondaryOptions) {
+		node.walk(({ node: childNode }) => {
+			if (!isFunctionNode(childNode)) {
+				return;
+			}
+
+			ignored = optionsMatches(secondaryOptions, 'ignoreFunctions', childNode.getName());
+
+			if (ignored) {
+				return false; // Stop iteration if an ignored function is found
```

Comment:
question

Can you have multiple functions per media query?
If so, can you add a test where you have an invalid function that follows an ignored function?

Just checking if you are not returning too early.
i.e. missing potential reports